src/audio.c

```diff
@@ -112,12 +114,10 @@ struct autx {
 	} stats;
 
 #ifdef HAVE_PTHREAD
-	union {
-		struct {
-			pthread_t tid;/**< Audio transmit thread           */
-			bool run;     /**< Audio transmit thread running   */
-		} thr;
-	} u;
+	struct {
+		pthread_t tid;/**< Audio transmit thread           */
+		bool run;     /**< Audio transmit thread running   */
+	} thr;
```

Comment:
this change is unrelated, I fixed it in master:

4b8a6d4180252ba7662f66da37feecb6fb1bbc31